sled-agent/src/services.rs

```diff
@@ -613,13 +645,21 @@ impl ServiceManager {
             // opte_ports=
             vec![],
             bootstrap_vnic,
-            link,
+            links.clone(),
             limit_priv,
         )
         .await?;
 
         let running_zone = RunningZone::boot(installed_zone).await?;
 
+        for link in &links {
+            info!(self.inner.log, "Ensuring {}/ll exists in zone", link.name(),);
+            Zones::ensure_has_link_local_v6_address(
+                Some(running_zone.name()),
+                &AddrObject::new(link.name(), "ll").unwrap(),
```

Comment:
Use `IPV6_LINK_LOCAL_NAME` from `illumos-utils/src/addrobj.rs`?

Reply:
Done in 85874f9a